app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
```

Comment:
@shir this is general base controller for different versions, I think it should be in api folder